pyuvsim/simsetup.py

```diff
@@ -1269,14 +1417,24 @@ def parse_time_params(time_params):
 
 def freq_array_to_params(freq_array):
     """
-    Give the channel width, bandwidth, start, and end frequencies corresponding
-    to a given frequency array.
+    Get a set of parameters that can be used to generate a given frequency array.
+
+    Returns a dictionary of parameters that can be used by parse_freq_params
```

Comment:
autoref `parse_freq_params`

Reply:
Actually it seems that there is no longer any such function. I'm just removing this sentence.